Tests/ComposableArchitectureTests/StoreTests.swift

```diff
@@ -435,6 +435,7 @@ final class StoreTests: XCTestCase {
     viewStore.send(0)
 
     XCTAssertEqual(emissions, [0, 3])
+  }
```

Comment:
Looks like the tests weren't building because of this 🤔

Reply:
Ohh, good catch. Broken merge I'd guess. Thank you!!